v21.2/use-changefeeds.md

```diff
@@ -98,9 +98,15 @@ The `kv.closed_timestamp.target_duration` [cluster setting](cluster-settings.htm
 
     To compare two different rows for [happens-before](https://en.wikipedia.org/wiki/Happened-before), compare the "updated" timestamp. This works across anything in the same cluster (e.g., tables, nodes, etc.).
 
-    Resolved timestamp notifications on every Kafka partition can be used to provide strong ordering and global consistency guarantees by buffering records in between timestamp closures. Use the "resolved" timestamp to see every row that changed at a certain time.
+    The complexity with timestamps is necessary because CockroachDB supports transactions that can affect any part of the cluster, and it is not possible to horizontally divide the transaction log into independent changefeeds. For more information about this, [read our blog post on CDC](https://www.cockroachlabs.com/blog/change-data-capture/) and see the following section on [resolved timestamps](#resolved-timestamps).
 
-    The complexity with timestamps is necessary because CockroachDB supports transactions that can affect any part of the cluster, and it is not possible to horizontally divide the transaction log into independent changefeeds. For more information about this, [read our blog post on CDC](https://www.cockroachlabs.com/blog/change-data-capture/).
+### Resolved timestamps
```

Comment:
I think we should add to the section that resolved messages are sent when we flush the buffer, and the resolved setting (aka resolved=10s) is an upper bound on how long it will take to flush that buffer - since this is a thing people often get confused about when it comes to resolved messages